net/net.sh

```diff
@@ -96,6 +98,7 @@ failOnValidatorBootupFailure=true
 genesisOptions=
 numFullnodesRequested=
 externalPrimordialAccountsFile=
+remoteExternalPrimordialAccountsFile=/tmp/external-primodial-accounts.yml
```

Comment:
Who populates /tmp/external-primodial-accounts.yml?   Perhaps `remoteExternalPrimordialAccountsFile` should be passed in as an argument to `net.sh`?

Reply:
The file is rsync-ed over from the calling machine (in this case, the CI buildkite VM), then its contents are cat'ed out to a known file that contains the internal and external node staking.  It's just a temp file to get it onto the correct machine, so no need to have a dynamic path for this one.